package.json

```diff
@@ -15,6 +15,7 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
+    "react-toastify": "^7.0.4",
```

Comment:
How much does this add to our payload?
If it's upward of 2-3KBs, please add a TODO comment in `index.tsx` to find an alternative later.

Reply:
It's 7kb gzipped we will add our custom toast later. Only using it because of time constraints.